### ISSUE TITLE
Using backtick

### ISSUE THREAD

**Issue by jonnieey** [adam-mcdaniel/dune#70]
Is it possible to use shell backtick  to expand variables eg.  ``vim `which COMMAND``

**Comment by Aloso** [comment]
No. Parameters are expanded by default if they are known variables:

```bash
vim os@name
# opens a file called "manjaro"
```

It should also work for programs:

```bash
vim (which COMMAND)
```

But that is currently broken. I'll open an issue.

**Comment by jonnieey** [comment]
Okay, thanks.

**Comment by jonnieey** [comment]
Now it stores variable as expected but still cannot work on them
eg
`let x = which mywork`
`echo x # /path/to/mywork`
`vim x # doesn't open the /path/to/mywork`
`vim (x) # doesn't open /path/to/mywork`
`vim (which mywork) # doesn't work`

I pulled from your repo Aloso/dune  branch  pipe-stdout

**Comment by Aloso** [comment]
I think the issue is that `which` adds a line break after the path.

**Comment by jonnieey** [comment]
Is there a workaround?

**Comment by jonnieey** [comment]
I even tried 
`vim (print (which mywork))`

which removes the newline but it doesn't work
